mods/default/locale/default.de.tr

```diff
@@ -11,6 +11,7 @@ Contents:=Inhalt:
 Save=Speichern
 by @1=von @1
 Page @1 of @2=Seite @1 von @2
+The book you were writing to mysteriously disappeared.=Das Buch, das Sie geschrieben haben, ist auf mysteriöse Weise verschwunden.
```

Comment:
Consider translating this as "Das Buch, an dem Sie geschrieben haben, ist auf mysteriöse Weise verschwunden". The current translation would translate to "The book you were writing ~~to~~ mysteriously disappeared.", which is subtly wrong.